descriptors/ModuleDescriptor-template.json

```diff
@@ -677,12 +677,15 @@
       "permissions": [
         "inventory-storage.instances.item.get",
         "inventory-storage.instances.item.put",
-        "mapping-metadata.get",
+        "mapping-metadata.item.get",
+        "mapping-metadata.type.item.get",
```

Comment:
The module uses only `/mapping-metadata/type/{recordType}` endpoint so "mapping-metadata.item.get" permission could be removed.